Patch: 79_archiver_slash.patch
Author: Lionel Elie Mamane <lionel@mamane.lu>
Some non-pipermail archivers require _no_ final slash (Closes: #350388)
Index: mailman-2.1.12/Mailman/Archiver/Archiver.py
===================================================================
--- mailman-2.1.12.orig/Mailman/Archiver/Archiver.py	2009-03-02 20:41:44.000000000 +0100
+++ mailman-2.1.12/Mailman/Archiver/Archiver.py	2009-03-02 20:50:25.000000000 +0100
@@ -146,8 +146,6 @@
                 'listname': self.internal_name(),
                 'hostname': hostname
                 }
-            if not url.endswith('/'):
-                url += '/'
             return url
 
     def __archive_file(self, afn):
Index: mailman-2.1.12/Mailman/Defaults.py.in
===================================================================
--- mailman-2.1.12.orig/Mailman/Defaults.py.in	2009-03-02 20:49:55.000000000 +0100
+++ mailman-2.1.12/Mailman/Defaults.py.in	2009-03-02 20:50:25.000000000 +0100
@@ -210,7 +210,7 @@
 # interpolate the name of the list into this.  You can also include a
 # "%(hostname)s" in the string, into which Mailman will interpolate
 # the host name (usually DEFAULT_URL_HOST).
-PUBLIC_ARCHIVE_URL = 'http://%(hostname)s/pipermail/%(listname)s'
+PUBLIC_ARCHIVE_URL = 'http://%(hostname)s/pipermail/%(listname)s/'
 
 # Are archives on or off by default?
 DEFAULT_ARCHIVE = On
